lib/PlaybackArea.tsx

```diff
@@ -148,6 +168,7 @@ export const PlaybackArea: React.FC<PlaybackAreaProps> = ({
   onPlaying,
   onSdp,
   metadataHandler,
+  secure,
```

Comment:
this should have a default value depending on `window.location.protocol`
```suggestion
  secure = window.location.protocol === 'https',
```